src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -890,11 +891,19 @@ void CompositionalMultiphaseBase::applySourceFluxBC( real64 const time,
                    "ElementRegions",
                    FieldSpecificationBase::viewKeyStruct::fluxBoundaryConditionString(),
                    [&]( FieldSpecificationBase const & fs,
-                        string const &,
+                        string const & setName,
                         SortedArrayView< localIndex const > const & targetSet,
                         Group & subRegion,
                         string const & )
   {
+    if( fs.getLogLevel() >= 2 && m_nonlinearSolverParameters.m_numNewtonIterations == 0 )
+    {
+      localIndex const numTargetElems = MpiWrapper::sum( targetSet.size() );
```

Comment:
Not that it matters for face BCs realistically, but for full correctness we should use the appropriate type to refer to global mesh quantities (we're making this mistake in a few more places I think): 
```suggestion
      globalIndex const numTargetElems = MpiWrapper::sum< globalIndex >( targetSet.size() );
```

Reply:
Good point, I made the modification that you suggested